kotlin-mui-material/src/jsMain/generated/mui/material/Select.kt

```diff
@@ -175,7 +175,7 @@ external interface SelectProps<Value> :
      * The variant to use.
      * @default 'outlined'
      */
-    var variant: SelectVariant?
+    var variant: SelectVariants?
```

Comment:
`SelectVariant`? Plural form isn't expected

Reply:
Single form expected for variant interface name (like in other components)

Reply:
Why? It's not expected.
In original MUI declarations it calls `SelectVariants`
Here is the usage https://github.com/mui/material-ui/blob/45cf002ee21d0b99f8ba2dc54e4d233499fb689b/packages/mui-material/src/Select/Select.d.ts#L151